deploy/backplane/acs/01-acs-admins-cluster.ClusterRole.yaml

```diff
@@ -98,6 +99,83 @@ rules:
   - get
   - list
   - watch
+# ACS SRE can view events
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can view pod logs
+- apiGroups:
+  - ""
+  resources:
+  - pods/log
+  verbs:
+  - get
+  - watch
+  - list
+# ACS SRE can view deployments, statefulsets, daemonsets, replicasets, replicationcontrollers
```

Comment:
Are you intent to add those perms into "backplane-acs-admin-cluster" cluster role? This will allow you to view all those resources including the customer's projects which we should not do.

Reply:
@wanghaoran1988 yes, that is correct. These are permissions for ACS-CS (all those clusters are fully internal). Please read https://github.com/openshift/managed-cluster-config/blob/master/docs/backplane/requirements/acs/10-acs-managed-service.md

Reply:
pinging @jewzaam

Reply:
Per [day-to-day guidelines](https://github.com/openshift/managed-cluster-config/blob/master/docs/backplane/guideline.md#day-to-day) it's acceptable to grant view on namespaced resources for RH managed systems like this one.